nsxt/resource_nsxt_policy_dhcp_v4_static_binding.go

```diff
@@ -68,24 +68,19 @@ func resourceNsxtPolicyDhcpV4StaticBinding() *schema.Resource {
 	}
 }
 
-func getPolicyDchpStaticBindingOnSegment(id string, segmentPath string, connector client.Connector, isGlobalManager bool) (*data.StructValue, error) {
+func getPolicyDchpStaticBindingOnSegment(context utl.SessionContext, id string, segmentPath string, connector client.Connector) (*data.StructValue, error) {
 	_, gwID, segmentID := parseSegmentPolicyPath(segmentPath)
-	if isGlobalManager {
-		client := gm_segments.NewDhcpStaticBindingConfigsClient(connector)
-		return client.Get(segmentID, id)
-	}
-	if gwID == "" {
-		// infra segment
-		client := segments.NewDhcpStaticBindingConfigsClient(connector)
+	if context.ClientType == utl.Global || gwID == "" {
```

Comment:
fixed segments might be supported on GM now. I suggest not to change the logic in this PR but lets check this later (it would simplify the code to only look at GW present in this condition)

Reply:
OK. So should we open an issue to track this?